gsa_admin.py

```diff
@@ -209,8 +209,9 @@ class gsaWebInterface:
   loggedIn = None
   _url_opener = None
 
-  def __init__(self, hostName, username, password, port=8000):
-    self.baseURL = 'http://%s:%s/EnterpriseController' % (hostName, port)
+  def __init__(self, hostName, username, password, port=8000, ssl=False):
```

Comment:
please change to a more descriptive var name like "use_ssl" to indicate this is a behavior modifier instead of an attribute.